templates/tutorial/tutorial/diff.html

```diff
@@ -38,25 +38,26 @@ <h1 {% if tutorial.image %}class="illu"{% endif %}>
 
 
 {% block content_ext %}
-    <h2>{% trans "Nouveaux Fichiers" %}</h2>
-    {% for add in path_add %}
-        {% with add_next=add.b_blob|repo_blob %}
-            <h3>{{ add.a_blob.path }}</h3>
-            <div class="diff_delta">
-                {{ ''|diff_text:add_next|safe }}
-            </div>
-        {% endwith %}
+    <h2>{% trans "Fichiers modifiés" %}</h2>
```

Comment:
`{% if modified %}` ne fonctionne pas car `modified` n'est pas une liste mais un itérateur.

Par contre le coup du empty fonctionne ! Je vais donc modifier. Les titres seront toujours présents par contre. Si qqn a une solution propre/performante pour tester si un itérateur est vide, je peux cacher les titres dans ce cas là mais il y a encore un autre problème: celui du pluriel/singulier.